src/openvic-simulation/map/ProvinceInstance.cpp

```diff
@@ -29,7 +33,7 @@ ProvinceInstance::ProvinceInstance(
 	event_modifiers {},
 	slave { false },
 	crime { nullptr },
-	rgo { nullptr },
+	rgo { { pop_type_keys} },
```

Comment:
```suggestion
	rgo { pop_type_keys },
```
No need for the double `{}`, in fact that forces the creation of a temporary intermediate object which is then either move or copy constructed into the actual variable.